lib/matplotlib/backends/backend_pgf.py

```diff
@@ -49,13 +51,31 @@
     except:
         warnings.warn('error getting fonts from fc-list', UserWarning)
 
+
+luatex_version_re = re.compile(
+    'This is LuaTeX, Version (?:beta-)?([0-9]+)\.([0-9]+)\.([0-9]+)'
+)
+
+
 def get_texcommand():
     """Get chosen TeX system from rc."""
     texsystem_options = ["xelatex", "lualatex", "pdflatex"]
     texsystem = rcParams["pgf.texsystem"]
     return texsystem if texsystem in texsystem_options else "xelatex"
 
 
+def get_lualatex_version():
```

Comment:
Note to other devs: would be nice to get #9571/#9639 merged first and integrate this version checking into that framework.

In the meantime, this function could(?) stay private.